src/app/environment.py

```diff
@@ -2,13 +2,20 @@
 Module dedicated to interacting with the environment (variables, version.json)
 """
 import json
+from enum import Enum
 from functools import lru_cache
 from pathlib import Path
 from typing import Optional
 
 from pydantic import AnyUrl, BaseSettings
 
 
+class Environment(str, Enum):
```

Comment:
👀 I like this more than https://github.com/mozilla/jira-bugzilla-integration/blob/a888e9fa080791e29b97e3284980108875c5186c/src/jbi/__init__.py#L8